### DISCUSSION TITLE
Will there be macOS support?

### DISCUSSION THREAD

**Discussion by voltangle** [davdroman/swiftui-navigation-transitions#74] [Q&A] [answered]
As in title, is there macOS support planned at least in the future?

**Comment by davdroman** [comment]
@ggoraa I know very little about custom transitions in macOS. Is there a similar AppKit API available like we have in UIKit?

**Reply by davdroman** [reply, marked as answer]
Upon further inspection, I don't believe there's an API in AppKit for this, so unless you're using Catalyst, it's not in the plans to support custom transitioning behavior on macOS.
